src/client/components/hud/characterHud/Conditions.tsx

```diff
@@ -44,6 +45,24 @@ export function ConditionsBar({ character }: { character: RRCharacterProps }) {
 
   return (
     <div className="flex flex-wrap flex-row-reverse pointer-events-auto">
+      {character.currentlyConcentratingOn && (
+        <div
+          className={clsx(
+            character.currentlyConcentratingOn.roundsLeft <= 1
+              ? "bg-red-700"
+              : character.currentlyConcentratingOn.roundsLeft <= 3
+              ? "bg-orange-500"
+              : "bg-gray-200",
+            "self-center select-none rounded-lg p-1 text-black ml-1"
+          )}
+          title={`${character.currentlyConcentratingOn.name}
+          \n${character.currentlyConcentratingOn.roundsLeft} rounds left`}
```

Comment:
You are using both `\n` and a line break here. Is that intended?

Reply:
Ah, I just didn't realize template strings preserved white spaces in them.